D3D12HelloWorld/Graphics/CommandList.cs

```diff
@@ -219,6 +265,10 @@ public sealed class CommandList : IDisposable {
                                        ulong intermediateOffset, int firstSubresource, ReadOnlySpan<byte> data)
             => mDevice.NativeDevice.UpdateSubresource(mCommandList, destResource.NativeResource, intermediateResource, intermediateOffset, firstSubresource, data);
 
+        public ulong UpdateSubresources(GraphicsResource destResource, ID3D12Resource intermediateResource,
+                                        ulong intermediateOffset, int firstSubresource, SubresourceInfo[] subresourceInfo)
+            => mDevice.NativeDevice.UpdateSubResources(mCommandList, destResource.NativeResource, intermediateResource, intermediateOffset, firstSubresource, subresourceInfo);
+
```

Comment:
Not used, but just exposing the other MemoryExtension overload via this abstraction to avoid the need for anyone wanting to use it to need to access the native command list directly, that can use this instead.